torch/csrc/autograd/python_variable.cpp

```diff
@@ -539,7 +539,7 @@ static PyMappingMethods THPVariable_as_mapping = {
 };
 
 static PyMethodDef extra_methods[] = {
-  {"_make_subclass", (PyCFunction)(void(*)(void))THPVariable_make_subclass, METH_STATIC | METH_VARARGS | METH_KEYWORDS, nullptr},
+  {"make_subclass", (PyCFunction)THPVariable_make_subclass, METH_STATIC | METH_VARARGS | METH_KEYWORDS, nullptr},
```

Comment:
I would be in favor of keeping both here. I am pretty sure some people use this outside our codebase.

Reply:
Done.